src/components/Accounts/Actions/PayManyModal.js

```diff
@@ -0,0 +1,399 @@
+    const dataOptions = payments.filter(p => p.ownerId === ownerId).map(p => ({ id: p.id, label: p.nameMethod }));
+    const ownerOptions = owners.map(o => ({ value: o.id, label: o.owner }));
+
+    const placeholderTranslationId = commentRequired
+      ? 'ui-users.accounts.pay.placeholder.additional.required'
+      : 'ui-users.accounts.pay.placeholder.additional.optional';
+
+    const submitButtonDisabled = pristine || submitting || invalid;
+    return (
+      <Modal
+        open={open}
+        label={<FormattedMessage id="ui-users.accounts.pay.payFeeFine" />}
+        onClose={onClose}
+        size="medium"
+        dismissible
+      >
+        <form>
+          <Row>
+            <Col xs>{this.renderFormMessage()}</Col>
+          </Row>
+          <br />
+          <Row>
+            <Col xs={4.5}>
+              <Row end="xs">
+                <Col xs={7} style={{ marginRight: '5px' }}>
+                  <FormattedMessage id="ui-users.accounts.pay.field.totalamount" />
+                </Col>
+                <Col xs={4} style={{ marginRight: '5px' }}>
+                  {parseFloat(totalAmount).toFixed(2)}
+                </Col>
+              </Row>
+              <Row end="xs">
+                <Col xs={7} style={{ marginRight: '5px' }}>
+                  <FormattedMessage id="ui-users.accounts.pay.field.selectedamount" />
+                </Col>
+                <Col xs={4}>
+                  {parseFloat(selected).toFixed(2)}
+                </Col>
+              </Row>
+              <Row end="xs">
+                <Col xs={7} style={{ marginRight: '10px' }}>
+                  <b><FormattedMessage id="ui-users.accounts.pay.field.paymentamount" /></b>
+                  :
+                </Col>
+                <Col xs={4} style={{ marginRight: '5px' }}>
+                  <div dir="rtl">
+                    <Field
+                      name="amount"
+                      component={TextField}
+                      onChange={this.onChangeAmount}
+                      hasClearIcon={false}
+                      fullWidth
+                      marginBottom0
+                      autoFocus
+                      required
+                    />
+                  </div>
+                </Col>
+              </Row>
+              <Row end="xs">
+                <Col xs={7}>
+                  <FormattedMessage id="ui-users.accounts.pay.field.remainingamount" />
+                </Col>
+                <Col xs={4}>
+                  {remaining}
+                </Col>
+              </Row>
+            </Col>
+            <Col xs={4}>
+              <Row><Col xs><FormattedMessage id="ui-users.accounts.pay.field.ownerDesk" /></Col></Row>
+              <Row>
+                <Col xs>
+                  <Field
+                    name="ownerId"
+                    component={Select}
+                    dataOptions={ownerOptions}
+                    placeholder="Select desk"
```

Comment:
I don't think we typically use placeholder text, but if we do provide it, certainly it would need to be provided as either a `<FormattedMessage>` prop, or, if the placeholder needs to be a string instead of an object, you'd have to wrap the `<Field>` in a `<FormattedMessage>` and use a render prop like: 
```
<FormattedMessage id='placeholder-id'>
  {placeholder => (
    <Field
      ...
      placeholder={placeholder}
    />
  )}
</FormattedMessage>
```
Likewise for the other placeholder props in this component. @JohnC-80, can you confirm whether we should or should not be provide placeholder text?